{
	"id": "CVE-2019-16768",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2019-16768",
				"title": "Internal exception message exposure for login action in Sylius",
				"description": "In affected versions of Sylius, exception messages from internal exceptions (like database exception) are wrapped by \\Symfony\\Component\\Security\\Core\\Exception\\AuthenticationServiceException and propagated through the system to UI. Therefore, some internal system information may leak and be visible to the customer. A validation message with the exception details will be presented to the user when one will try to log into the shop. This has been patched in versions 1.3.14, 1.4.10, 1.5.7, and 1.6.3.",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "GitHub_M",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:L/I:N/A:N",
							"base_score": 3.5,
							"base_severity": "LOW",
							"temporal_score": 3.5,
							"temporal_severity": "LOW",
							"environmental_score": 3.5,
							"environmental_severity": "LOW"
						}
					}
				],
				"cwe": [
					{
						"source": "GitHub_M",
						"cwe": [
							"CWE-209"
						]
					}
				],
				"references": [
					{
						"source": "CVE",
						"url": "https://github.com/Sylius/Sylius/commit/be245302dfc594d8690fe50dd47631d186aa945f"
					},
					{
						"source": "CVE",
						"url": "https://github.com/Sylius/Sylius/security/advisories/GHSA-3r8j-pmch-5j2h"
					},
					{
						"source": "GitHub_M",
						"url": "https://github.com/Sylius/Sylius/commit/be245302dfc594d8690fe50dd47631d186aa945f"
					},
					{
						"source": "GitHub_M",
						"url": "https://github.com/Sylius/Sylius/security/advisories/GHSA-3r8j-pmch-5j2h"
					}
				],
				"modified": "2024-08-05T01:24:47.238Z"
			}
		}
	],
	"data_source": "mitre-v5"
}
